support/Pipelines/Bbh/Ringdown.yaml

```diff
@@ -45,10 +45,26 @@ DomainCreator:
 
 Evolution:
   InitialTime: 0.0
-  InitialTimeStep: 0.01
+  InitialTimeStep: 0.0002
+  # This is the smallest interval we'd need to observe time step/constraints. If
+  # you need it smaller you can edit it, but make sure to change the slab
+  # intervals in the EventsAndTriggers
+  InitialSlabSize: 0.5
```

Comment:
Most of the observations in this file are based on the slab size, so you've decreased the amount of output by a factor of 50 here.  For the constraint norms that might be fine, but it's probably not what you want for the actual data observations.

Reply:
This was taken to match the inspiral.yaml and I assumed we'd want a similar amount of output during the ringdown.

Reply:
I don't know.  The dynamical timescales are shorter, so maybe people want more data?  @geoffrey4444 or someone who actually analyzes runs want to weigh in?

Reply:
Here is what I actually did in the ringdown that worked; I'd suggest something similar in the pipeline:
```yaml
Evolution:
  InitialTime: 0.0
  InitialTimeStep: 0.0002
  InitialSlabSize: 0.1
  StepChoosers:
    - Increase:
        Factor: 2
    - ErrorControl:
        AbsoluteTolerance: 1e-10
        RelativeTolerance: 1e-8
        MaxFactor: 2
        MinFactor: 0.25
        SafetyFactor: 0.95
  TimeStepper:
    AdamsBashforth:
      Order: 4
```

Reply:
I don't have any actual successful ringdown runs with global time stepping, so I don't have an opinion on good choices for that case

Reply:
How often do you do observations?  That's more what I'm worried about here.

Reply:
I don't think we know exactly how often we want to do observations. `0.1` for a slab size seems reasonable for now. We'll probably have to revisit this later though as we do more testing and actually do a successful ringdown with the pipeline. I think the more important part of this PR is the switch to LTS itself.